myuw/data/resource_link_import.csv

```diff
@@ -12,7 +12,7 @@ Academics,Advising & Tutoring,all,https://grad.uw.edu/for-students-and-post-docs
 Academics,Grades & Transcripts,all,https://www.washington.edu/uaa/advising/academic-planning/plan-your-degree/,GPA Calculator,,,,,,,no
 Academics,Grades & Transcripts,all,https://sdb.admin.uw.edu/sisStudents/uwnetid/grades.aspx,Grade Report,,,,,,,no
 Academics,Grades & Transcripts,all,,,,,http://www.uwb.edu/registration/transcript,About Ordering Transcripts,,,no
```

Comment:
This URL (currently only shown to UWB) redirects to https://registrar.washington.edu/students/transcripts/. Can we go ahead and update this so that it shows to all campuses?